SPECS/emacs/emacs.spec

```
@@ -0,0 +1,86 @@
+Summary:        GNU Emacs text editor
+Name:           emacs
+Version:        28.1
+Release:        2%{?dist}
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+License:        GPLv3+ and CC0-1.0
+URL:            http://www.gnu.org/software/emacs/
+Group:          Applications/Editors
+Source0:        https://ftp.gnu.org/gnu/emacs/emacs-%{version}.tar.xz 
+
+BuildRequires:  gcc
+BuildRequires:  glibc-devel
+BuildRequires:  ncurses-devel
+BuildRequires:  systemd-devel
+BuildRequires:  gnutls-devel
+
+%description
+Emacs is a powerful, customizable, self-documenting, modeless text
+editor. Emacs contains special code editing features, a scripting
+language (elisp), and the capability to read mail, news, and more
+without leaving the editor.
+
+%prep
+%autosetup
+
+%build
+%configure \
+            --without-xpm            \
+            --without-jpeg           \
+            --without-tiff           \
+            --without-gif            \
+            --without-png            \
+            --without-rsvg           \
+            --without-lcms2          \
+            --without-xft            \
+            --without-harfbuzz       \
+            --without-m17n-flt       \
+            --without-toolkit-scroll-bars \
+            --without-xaw3d          \
+            --without-xim            \
+            --without-makeinfo
+make %{?_smp_mflags}
+
+%install
+make DESTDIR=%{buildroot} %{?_smp_mflags} install
```

Nit: Can use macros here
```suggestion
%make_install
```

---

updated.